File: app/src/main/java/com/eagskunst/emmanuel/gamingnews/models/Game.java
package com.eagskunst.emmanuel.gamingnews.models;

import com.google.gson.annotations.SerializedName;


public class Game{

	@SerializedName("cover")
	private Cover cover;

	@SerializedName("name")
	private String name;

	@SerializedName("id")
	private int id;

	@SerializedName("url")
	private String gameUrl;

	public void setCover(Cover cover){
		this.cover = cover;
	}

	public Cover getCover(){
		return cover;
	}

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setId(int id){
		this.id = id;
	}

	public int getId(){
		return id;
	}

	public String getGameUrl() {
		return gameUrl;
	}

	public void setGameUrl(String gameUrl) {
		this.gameUrl = gameUrl;
	}

    @Override
    public String toString() {
        return "Game{" +
                "cover=" + cover +
                ", name='" + name + '\'' +
                ", id=" + id +
                ", gameUrl='" + gameUrl + '\'' +
                '}';
    }
}

File: app/src/main/java/com/eagskunst/emmanuel/gamingnews/models/Game.kt
package com.eagskunst.emmanuel.gamingnews.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Game {

    @SerializedName("cover")
    @Expose
    var cover: Cover? = null

    @SerializedName("name")
    var name: String? = null

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("url")
    var gameUrl: String? = null

    override fun toString(): String {
        return "Game{" +
                "cover=" + cover +
                ", name='" + name + '\'' +
                ", id=" + id +
                ", gameUrl='" + gameUrl + '\'' +
                '}'
    }
}
